components/badge.css

```diff
@@ -0,0 +1,73 @@
+.badge-container {
+    position: relative;
+    width: fit-content;
+    box-sizing: border-box;
+
+}
+
+.status-badge {
+    height: 1.1rem;
+    width: 1.1rem;
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    position: absolute;
+    font-size: 0.8em;
+    color: white;
+    padding: 0.5rem;
+}
+
+
+.online {
+    background-color: var(--green);
+}
+
+.offline {
+    background-color: var(--red);
+}
+
+.dont-disturb {
+    background-color: var(--orange);
+}
+
+
```

Comment:
always give single line space after one class/ID/Tag styling. that will look good.

Reply:
noted !! removed all the extra spaces.